src/V3Timing.cpp

```diff
@@ -57,15 +60,17 @@
 VL_DEFINE_DEBUG_FUNCTIONS;
 
 // ######################################################################
-//  Transform nodes affected by timing
+//  Detect nodes affected by timing
 
-class TimingVisitor final : public VNVisitor {
+class SuspendableVisitor final : public VNVisitor {
```

Comment:
```suggestion
class TimingSuspendableVisitor final : public VNVisitor {
```

Generally I prefer visitor classes should start with the name of the C++ file to help readers.

(The Vertex probably should have likewise had that but it escaped so is grandfathered.)